Makefile

```diff
@@ -84,6 +86,10 @@ test-all: clean-test docker-images
 	tox -- -s tests.integration
 	tox -e py26 -- -s ${TEST_SUITE} -a '!quarantine'
 
+IMAGE_NAMES?="all"
```

Comment:
if the variable hasn't been exported, it will default to setting it to "all". Otherwise it uses the exported value